webform_civicrm.module

```diff
@@ -28,6 +28,7 @@ define('WEBFORM_CIVICRM_DEFAULT_CONTACT_ID', 1);
  */
 function webform_civicrm_library_info_alter(array &$libraries, $extension) {
   if ($extension === 'webform_civicrm') {
+    \Drupal::getContainer()->get('civicrm')->initialize();
```

Comment:
non blocker, but probably best to write it as `\Drupal::service('civicrm')->initialize();`?

Reply:
I don't understand what's best here -> I just looked at examples from code that I know MattG has touched: 

![image](https://user-images.githubusercontent.com/5340555/83313062-e0cd2900-a1d1-11ea-8fa8-d83c4951e751.png)

If technically `\Drupal::service('civicrm')->initialize();` is better -> happy to follow your lead on that.

Reply:
pretty sure both are same technically, service() internally calls the same function `getContainer()->get($id)`, but seems the latter is slightly better to read :).

Reply:
Ok - happy to go with that!